### ISSUE TITLE
Plugin Order of Execution

### ISSUE THREAD

**Issue by JohnMcLear** [ether/etherpad-lite#2005]
TLDR; ep_file_menu plugin needs to run before other plugins and expose a new hook to other plugins

Problem:  I need to expose a Hook from inside a plugin so other plugins to be able to provide HTML to the hook.  Think of it like how the eejsBlock_XX hooks work right now, well I need a plugin to expose an eejsBlock other plugins can write to..  

I know it's possible to have hooks execute in a specific order but I'd need to ensure my plugin is executed before any other plugin that wants to write to this hook.

I haven't tried anything yet or tested anything, I just wanted to document something here for posterity..
